website/package.json

```diff
@@ -30,14 +31,31 @@
     "url-loader": "^4.1.1"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^13.5.0",
+    "babel-jest": "^27.5.1",
     "css-loader": "^3.4.2",
     "dotenv": "^10.0.0",
+    "jest": "^27.5.1",
+    "jest-cli": "^27.5.1",
     "path-browserify": "^1.0.1",
     "process": "^0.11.10",
     "stream-http": "^3.2.0",
     "style-loader": "^1.1.3",
     "svg-inline-loader": "^0.8.2",
     "tty-browserify": "0.0.1"
   },
+  "jest": {
+    "transform": {
+      "^.+\\.(js|jsx)$": "babel-jest"
```

Comment:
It looks like Docusaurus uses [SWC](https://swc.rs/) instead of Babel [for its transformation](https://github.com/facebook/docusaurus/blob/main/jest.config.mjs#L42)? [This looks like it was a very recent change](https://github.com/facebook/docusaurus/pull/6944). There could be minor differences in transformation output, which could mean the component we test is different than what's in production. If so, I think we should try to use SWC as well?

Reply:
That makes sense, I will try switching that out and removing the babel config for a similar config as Docusaurus. I appreciate the input with this!